tests/ti_deps/deps/test_trigger_rule_dep.py

```diff
@@ -151,7 +151,7 @@ def test_one_failure_tr_success(self, get_task_instance):
         dep_statuses = tuple(
             TriggerRuleDep()._evaluate_trigger_rule(
                 ti=ti,
-                successes=0,
+                successes=1,
```

Comment:
Why do we need to change test of another trigger rule?

Reply:
I think by mistake I made this change. I have reverted it. thanks for pointing it out.